### ISSUE TITLE
a11y: Voice message play button in replies not reachable for Talkback screenreaders

### ISSUE THREAD

**Issue by fcnjd** [vector-im/element-android#8633]
### Steps to reproduce

1: Go into an existing chat with at least one message
2: Make a reply to a message by recording a voice message
3: Turn on Talkback screenreader

#### What did you expect?
I expect to find the play button for the voice message, by swiping to the left and right, because this is the classic way to move between Elements.
#### What happened instead?
Play button isn't reachable to Talkback users.
Sorry, I already submitted the logs by shaking the phone, and therefore didn't include the issue number. Next time I'll be smarter and create the issue first.

### Your phone model

Google Pixel 7

### Operating system version

13

### Application version and app store

1.6.5 [40106052] (F-fd6a45a3) isntalled from F-Droid

### Homeserver

Synapse 1.90.0

### Will you send logs?

Yes

### Are you willing to provide a PR?

No
